### PR TITLE
S3Bucket infrastructure setup

### PR DESCRIPTION
**Pull request by ctperry0301**
Hope I did this right haha

### CODE REVIEW

**Review by jpoist97 — COMMENTED** [review]
This looks right, before I approve it I was curious did you actually deploy to the aws account? Also something else to check before we merge this in is after you deploy the bucket can we put an image in the bucket and get a link from it to publicly view that image since we want to just pass that link into the business preview component

**Comment by ctperry0301** [comment]
It should now be deployed to the AWS account, I think when Rohith deployed his changes it deleted mine. 

And in terms of whether or not an image can be publicly viewed, that is not associated with an S3 bucket. Currently, the bucket is publicRead, but you still wont be able to view all images unless when you store the image as public-read as well. So this will work as long as you include the following code when an image is sent to S3

const s3Params = {
    Bucket: process.env.AWS_BUCKET_NAME,
    Key: `${fileName}.${fileType}`,
    Body: req.file.buffer,
    ACL: "public-read", // This line needs to be included when uploading a pic to the bucket so it can be seen. 
  };

**Review by jpoist97 — APPROVED** [review]
Great, looks good

**Pull request merged**
